acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/SignerConfiguration.java

```diff
@@ -32,6 +32,7 @@
   private Optional<TlsOptions> serverTlsOptions;
   private Optional<TlsCertificateDefinition> overriddenCaTrustStore;
   private final int metricsPort;
+  private final boolean slashingProtectionEnabled;
```

Comment:
you can probably just put in a URL - if its not null, then slashing is enabled, and you can set the url at the same time - I think I have this in one of my PRs?

Reply:
removing it, don't need to disable slashing protection as still using noop slashing protection. will just use the url when we merge in real slashing protection